src/main/java/com/excella/gradle/cucumber/CucumberRunner.java

```diff
@@ -50,7 +50,7 @@ public void runCucumberTests(JavaExec cucumberExec,
 
         if (formats != null) {
             for(String format : formats){
-                args.add("--format");
+                args.add("--plugin");
```

Comment:
Sort of why i up'ed the dependency version, but i will see if theres a way to make it work.
the old format still works but we get warnings so it may be removed at some point the future